data_processor.py

```diff
@@ -1,60 +1,55 @@
 import os,sys,csv
 
 class DataProcessor(object):
-    '''The constructor of the DataProcessor'''
-    def __init__(self, input_file):
-        self.input_file = input_file
-
-    '''load_file method loads the file from current folder and check if it's valid , with open will close file if any excetption happed '''
-    def load_file(self):
-        with open(self.input_file) as f:
-            data=f.readlines()
-            for line in data:
-                print(line)
+
+    '''load_file method loads the file from the current folder and check if it's valid , with open will close file if any
+    excetption happend '''
+    def load_file(self, input_file):
+        with open(input_file) as csv_file:
```

Comment:
you may need to check if file exists before try to open a file.
reference: https://stackoverflow.com/questions/82831/how-do-i-check-whether-a-file-exists